### PR TITLE
French translation Document Ready Script Tag jQuery

### PR DESCRIPTION
**Pull request by Tagada85**
French translation of the first jQuery Challenge (Document Ready and script tag).
I will do more later.

### CODE REVIEW

**Comment by atjonathan** [comment]
@Tagada85, We need someone in the [TranslationFrench](https://gitter.im/FreeCodeCamp/TranslationFrench) chat room to confirm or critique this translation. Are you planning to add more translations to this pull request, if so I'll mark it as `in progress`.

**Comment by Tagada85** [comment]
Added challenges : 
-Target HTML elements using jQuery
-Target Elements By Class using jQuery
-Target Elements By ID using jQuery

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 1]
```diff
@@ -69,6 +69,16 @@
...
         "Wir werden später noch mehr über <code>Funktionen</code> lernen. Wichtig zu wissen ist, dass der Code innerhalb der <code>Funktion</code> ausgeführt wird, sobald der Browser die Seite geladen hat.",
         "Das ist wichtig, denn ohne deine <code>Document Ready Funktion</code> könnte dein Code ausgeführt werden, bevor das HTML gerendert wurde - was zu Fehlern führen kann."
+      ],
+      "titleFr" : "Apprendre comment fonctionnent les balises Script et Document Ready",
+      "descriptionFr": [
+        "Nous sommes maintenant prêt à apprendre jQuery, l'outil Javascript le plus populaire de tout les temps. Ne vous inquiétez pas à propos de JavaScript lui-même, nous y viendrons bientôt.",
+        "Avant de pouvoir utiliser jQuery, nous avons besoin d'ajouter certaines choses à notre HTML.",
+        "Premièrement, ajoutez un élement <code>script</code> en haut de votre page. Assuez-vous de le fermer à la ligne suivante.",
```
Comment (mizujin) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Small typo on "Assuez-vous" Should read "Assurez-vous"

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 2]
```diff
@@ -69,6 +69,16 @@
...
+        "Votre navigateur exécutera JavaScript à l'intérieur de l'élement <code>script</code>, y compris jQuery.",
+        "'A l'intérieur de votre élement <code>script</code>, ajoutez ce code : <code>$(document).ready(function() {</code> à votre <code>script</code>. Puis fermez-le à la ligne suivante (toujours à l'intérieur de votre élement <code>script</code> avec: <code>});</code>",
```
Comment (mizujin) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Small typo on " 'A " should read " À "

**Comment by Tagada85** [comment]
Thank you @mizujin  !

**Comment by mizujin** [comment]
No problems @Tagada85 . I'll check your PR more in details shortly. it seems like you and I are the only currently active french translators, so if you don't mind checking my PR it'd be nice ! #9471

**Comment by mizujin** [comment]
@atjonathan I confirm this translation to be correct !

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

**Pull request merged**